kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -420,11 +427,19 @@ def exec_run(
 
 def exec_foundry_view_kcfg(foundry_out: Path, test: str, **kwargs: Any) -> None:
     kcfgs_dir = foundry_out / 'kcfgs'
+    contract_name = test.split('.')[0]
     foundry = Foundry(foundry_out)
     kcfg = KCFGExplore.read_cfg(test, kcfgs_dir)
     if kcfg is None:
         raise ValueError(f'Could not load CFG {test} from {kcfgs_dir}')
-    viewer = KCFGViewer(kcfg, foundry.kevm, node_printer=foundry.kevm.short_info)
+
+    def _short_info(cterm: CTerm) -> Iterable[str]:
+        return foundry.short_info_for_contract(contract_name, cterm)
+
+    def _custom_view(elem: KCFGElem) -> Iterable[str]:
+        return foundry.custom_view(contract_name, elem)
```

Comment:
This is fine as it is, but if you want to make this a bit more concise you can use `functools.partial`.

Reply:
I think I like the explicit look here.